### PR TITLE
feat: badge and button added.

### PR DESCRIPTION
**Pull request by krishnakant01**
I have added badges and buttons components. The website is not mobile responsive yet.

### CODE REVIEW

**Comment by krishnakant01** [comment]
Live demo: https://culer-ui.netlify.app/docs.html

**Comment by tusharkandpal** [comment]
code is well structured & easy to read.. UI looks damn good !! 💯

File: components/button.css [hunk 1]
```diff
@@ -0,0 +1,315 @@
...
+    color: var(--color-barcelona-blue);
+    padding: 0.75rem 1rem;
+    transition: all 0.3s ease-in-out;
+    text-decoration: none;
+}
+.btn-outlined-secondary:hover{
+    background-color: var(--color-medium-blue);
+    cursor: pointer;
+}
+
```
Comment (JantuDeb) [inline components/button.css]: Use a `btn` class for common properties, like, `padding`, `text-decoration`, `cursor`, etc..You can reduce your CSS in this way.

**Review by JantuDeb — CHANGES_REQUESTED** [review]
I have mentioned the issues you have in your CSS. Try to resolve this then you can merge.

File: components/button.css (hunk 1 above)
Comment (krishnakant01) [reply components/button.css]: Yes, I'll do that thanks.

**Pull request merged**